### PR TITLE
Walletimporthandler

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/e02wqk2w5dqvr5nmp8e4zjyg00

[download apk](http://filestorage.bluewallet.io:1488/1577336876.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/u1mw1c3e24jagjb78ebuvn130r

[download apk](http://filestorage.bluewallet.io:1488/1577339644.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wwp5mxnchgrndh8ve9wt5tzbew

[download apk](http://filestorage.bluewallet.io:1488/1577341182.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pktd27yuevkg490avdtaz4ed8g

[download apk](http://filestorage.bluewallet.io:1488/1577379471.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zu3kxvkduj1c6veqg9w2kqgutm

[download apk](http://filestorage.bluewallet.io:1488/1577381456.apk)

**Comment by Overtorment** [comment]
@marcosrdz  conflicts

**Comment by marcosrdz** [comment]
> @marcosrdz conflicts

Fixed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/eyaw4enerr2q9j2442w7x0yxdc

[download apk](http://filestorage.bluewallet.io:1488/1577414045.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pw5px871wvxmvenxez0aahy104

[download apk](http://filestorage.bluewallet.io:1488/1577479413.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9yq4a3kdgn082whqp29zv6w248

[download apk](http://filestorage.bluewallet.io:1488/1577479865.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/btxyqe1t4pfxpqvx72j3b385ur

[download apk](http://filestorage.bluewallet.io:1488/1577481960.apk)

File: class/abstract-wallet.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export class AbstractWallet {
     this.chain = Chain.ONCHAIN;
     this.hideBalance = false;
     this.userHasSavedExport = false;
+    this.use_with_hardware_wallet = false;
```
Comment (Overtorment) [inline class/abstract-wallet.js]: why is it here?

File: screen/lnd/scanLndInvoice.js [hunk 2]
```diff
@@ -122,7 +122,7 @@ export default class ScanLndInvoice extends React.Component {
       let w = this.state.fromWallet;
       let decoded;
       try {
-        decoded = await w.decodeInvoice(data);
+        decoded = w.decodeInvoice(data);
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js]: why `await` is lost?

Comment (marcosrdz) [reply screen/lnd/scanLndInvoice.js]: Ooops. Must've been from stashed changes when working on another PR

File: class/abstract-wallet.js (hunk 1 above)
Comment (marcosrdz) [reply class/abstract-wallet.js]: Figured the base class would want it. removed.

**Pull request merged**